README.md

```diff
@@ -1,703 +1,48 @@
-<p align="center">
-  <img src="https://github.com/lucasbento/create-graphql/raw/master/content/logo.png">
-</p>
+# ![Create-GraphQL Logo](/media/logo.png)
 
-<h1 align="center">Create GraphQL</h1>
+# Create-GraphQL
```

Comment:
Elements in the center are great, but I prefer to use markdown because its small and more clear. And I think that sounds better to keep accordin this way: [react-native#README](https://github.com/facebook/react-native#react-native---). What do you think?